modules/core/src/main/java/org/apache/synapse/config/xml/EnrichMediatorFactory.java

```diff
@@ -296,6 +296,25 @@ private int convertTypeToInt(String type) {
         return -1;
     }
 
+    private int convertActionToInt(String action) {
```

Comment:
Why do we need to convert enums to ints

Reply:
I followed the previous style used in the code ( there's a similar func for source/target types called convertTypeToInt )